### DISCUSSION TITLE
Asynchronous Import doesn't work

### DISCUSSION THREAD

**Discussion by oyzhen** [alephjs/aleph.js#166] [Q&A]
aleph version 0.3.0-alpha.8
deno version 1.7.5

Module '"https://deno.land/x/aleph@v0.3.0-alpha.8/mod"' has no exported member 'Import'

![image](https://user-images.githubusercontent.com/52534412/109393921-503d9800-795f-11eb-9bd5-faf8d26a050a.png)

**Comment by shadowtime2000** [comment]
0.3.0 is in alpha and is extremely unstable so we haven't updated the docs yet. Please check #5 for a guide on explaining the smart JSX transform.

**Reply by oyzhen** [reply]
But, how can I disable SSR for specific components like "Slate"?

**Reply by shadowtime2000** [reply]
@oyzhen I have to check again, but I don't think that is actually possible anymore.

**Reply by ije** [reply]
the import component has been removed in 0.3, please use react.lazy instead

**Reply by ije** [reply]
next patch the bundler also can work fine with `react.lazy`

**Reply by oyzhen** [reply]
OK, I'll try. Thanks.
